Makefile.uk

```diff
@@ -89,6 +89,8 @@ LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-missing-field-initializers
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-implicit-fallthrough
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-char-subscripts
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-sign-compare
+LIBPYTHON3_SUPPRESS_FLAGS-$(call have_gcc) += -Wno-address
+LIBPYTHON3_SUPPRESS_FLAGS-$(call have_clang) += -Wno-tautological-pointer-compare
```

Comment:
I would firsly put all items related to gcc, then all items related to clang. So, I would move the clnag line 3 lines below